tools/devtool

```diff
@@ -723,6 +723,7 @@ cmd_shell() {
 }
 
 cmd_sh() {
+    ensure_build_dir
```

Comment:
nit: We might also want to ensure devctr by using `ensure_devctr` like cmd_shell does.

Reply:
Interesting... Docker will actually auto-pull, so this all works ok if Docker is present. That function does check if Docker exists before running it.

Reply:
Will keep it in mind for the future.